lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -141,6 +145,21 @@ class DetailsRenderer {
   }
 
   /**
+   * @param {!DetailsRenderer.NodeDetailsJSON} item
+   * @return {!Element}
+   */
+  _renderNode(item) {
```

Comment:
node is slightly ambiguous at this level, can we make it DOMNode or something? when I just looked at the title of the PR I thought it was a renderer running in "node.js" 😆